pkg/volume/csi/labelmanager/labelmanager.go

```diff
@@ -153,7 +153,7 @@ func verifyAndAddNodeId(
 			string(jsonObj))
 		_, updateErr := k8sNodesClient.Update(result)
 		if updateErr == nil {
-			fmt.Printf(
+			glog.V(1).Infof(
```

Comment:
V(1) seems a little questionable.  @saad-ali says this path should only be called when a driver first registers, and even if the driver crashes and restarts, if it's not recreating it's socket then it should not come here.  Given that, I think V(2) would be better, and we should change all the V(1) to V(2).